File: benchmark/perl/strcat.pl
use Time::HiRes qw(time);
my $s = "";
my $start = time();
$s .= "x" for 1..50000;
my $end = time();
my $ms = ($end - $start) * 1000;
print "Length: " . length($s) . "
";
printf "Time: %.0fms
", $ms;

File: benchmark/src/php/strcat.php
<?php
$s = "";
$start = hrtime(true);
for ($i = 0; $i < 50000; $i++) { $s .= "x"; }
$end = hrtime(true);
$ms = ($end - $start) / 1e6;
echo "Result: " . strlen($s) . "
";
echo "Time: " . round($ms) . "ms
";
